src/middlewares/app-auth-manager.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * provides credentials to this web app
+ */
+
+require('dotenv').config()
+const assert = require('node:assert/strict')
+
+const basicAuth = require('express-basic-auth')
+
+const create = logger => {
+  assert.notEqual(process.env.APP_USER_NAME, undefined, 'APP_USER_NAME env variable is mandatory to run this app')
+  const username = process.env.APP_USER_NAME
```

Comment:
Consider a proper, non-versioned config file option, instead of environment variables.